staging/src/k8s.io/kube-gen/cmd/informer-gen/generators/customargs.go

```diff
@@ -22,10 +22,12 @@ type CustomArgs struct {
 	VersionedClientSetPackage string
 	InternalClientSetPackage  string
 	ListersPackage            string
+	SingleDirectory           bool
 }
 
 func (ca *CustomArgs) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&ca.InternalClientSetPackage, "internal-clientset-package", ca.InternalClientSetPackage, "the full package name for the internal clientset to use")
 	fs.StringVar(&ca.VersionedClientSetPackage, "versioned-clientset-package", ca.VersionedClientSetPackage, "the full package name for the versioned clientset to use")
 	fs.StringVar(&ca.ListersPackage, "listers-package", ca.ListersPackage, "the full package name for the listers to use")
+	fs.BoolVar(&ca.SingleDirectory, "single-directory", ca.SingleDirectory, "if false, will output to \"internalversion\" and \"externalversion\" subdirecotries")
```

Comment:
Better: if true, omit the intermediate \"internalversion\" and \"externalversion\" subdirs